File: 1A_Theatre_Square.cpp
// CodeForces
// Theatre Square 
// GNU C++ 17 

#include <iostream>
using namespace std; 

int main()
{
    long long n = 0, m = 0, a = 0;
    cin >> n >> m >> a; 

    long long x = n / a; 
    long long y = m / a; 
    if(n % a != 0) x++; 
    if(m % a != 0) y++; 
    cout << x*y << endl; 
}

File: 1A_Theatre_Square.py
#CodeForces 
#Theatre Square     
#Python 3.6.5 

import sys 

userInput = input()
n, m, a = userInput.split()

n = int(n)
m = int(m)
a = int(a) 

x = n // a 
y = m // a 
if n % a != 0: 
    x += 1 
if m % a != 0: 
    y += 1 
print(x*y) 

